src/temp/ProjectListControls/ViewModeTabs.tsx

```diff
@@ -0,0 +1,71 @@
+import themeProperties from '../../theme/themeProperties';
+import { useState, ReactNode } from 'react';
+import style from './Search.module.scss';
+import ListIcon from '../icons/ListIcon';
+import LocationIcon from '../icons/LocationIcon';
+import { SetState } from '../../features/common/types/common';
+
+interface ViewModeTabsProps {
+  setIsFilterOpen: SetState<boolean>;
+  isSearching: boolean;
+}
+
+type Tabs = 'list' | 'map';
+interface TabItemProps {
+  selectedTab: Tabs;
+  icon: ReactNode;
+  label: Tabs | undefined;
+}
+const ViewModeTabs = ({ setIsFilterOpen, isSearching }: ViewModeTabsProps) => {
+  const { dark, light } = themeProperties;
+  const [selectedMode, setSelectedMode] = useState<'list' | 'map'>('list');
+  const selectTab = (tab: Tabs) => {
+    setIsFilterOpen(false);
+    setSelectedMode(tab);
+  };
+
+  const TabItem = ({ selectedTab, icon, label }: TabItemProps) => {
+    const isSelected = selectedMode === selectedTab;
+    const tabButtonClass = isSelected
+      ? style.selectedTabButton
+      : style.unselectedTabButton;
+    return (
+      <button className={tabButtonClass} onClick={() => selectTab(selectedTab)}>
+        {icon}
+        <div className={style.label}>{label}</div>
+      </button>
+    );
+  };
+
+  const tabContainerClass = isSearching
+    ? style.tabContainerSecondary
+    : style.tabContainer;
+
+  const getIconColor = (mode: Tabs, selectMode: Tabs) =>
+    mode === selectMode ? light.light : dark.darkNew;
+
+  return (
+    <div className={tabContainerClass}>
+      <TabItem
+        selectedTab="list"
+        icon={
+          <ListIcon width={14} color={getIconColor('list', selectedMode)} />
+        }
+        label={isSearching ? undefined : 'list'}
+      />
+      <TabItem
+        selectedTab="map"
+        icon={
+          <LocationIcon
+            width={9}
+            height={13}
```

Comment:
1. Why provide both height and width?
2. For uniformity, height of the 2 tab icons should match.